ExampleGenerator/Series/StemSeriesExamples.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace ExampleGenerator
+{
+	using OxyPlot;
+	using OxyPlot.Axes;
+	using OxyPlot.Series;
+
+	public class StemSeriesExamples
+	{
+		[Export(@"Series\StemSeries")]
+		public static PlotModel StemSeries()
+		{
+
+			var model = new PlotModel{ Title = "Fun with Bats" };
+
+			Func<double, double>[] batFn = new Func<double, double>[]
```

Comment:
I think the functions are complicating the example here. Can you find some other discrete sequence data to demonstrate stem series? http://se.mathworks.com/help/matlab/ref/stem.html